docs/environment-variables.md

```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
 export FLASK_ENVIRONMENT_CONFIG=dev
 ```
 
-Windows user should use **set** instead of **export** for setting these environment variables. <br/>
```

Comment:
No need to remove this, this line in intentional to help Windows users. Can you undo this change, please?